src/app/dogs/dogs.component.html

```diff
@@ -6,11 +6,11 @@
 
 <ul style="list-style-type: none;" href="#" class="nav-link link-dark">
     <ng-container *ngFor="let breed of filteredDogs">
-        <li class="caret" [routerLink]="['/breed',breed.breedName]">
-            {{ breed.breedName | titlecase}}
-            <ul class="nested" *ngFor="let subBreed of breed.subBreedNames">
+        <li [appHighlight]="searchText" [filtredList]="breed.breedName" [routerLink]="['/breed',breed.breedName]" style="text-transform:capitalize;">
+            {{ breed.breedName }}
```

Comment:
```suggestion
        <li [appHighlight]="searchText" [filtredList]="breed.breedName" [routerLink]="['/breed',breed.breedName]">
            {{ breed.breedName | titlecase }}
```
did you try this approach?